### PR TITLE
[RFR] Load config from .comfy/config when available

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: admin/bin/comfy-admin.js [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
+    quiet: true,
+});
+
+webpackServer.listen(port, error => {
     if (error) {
         console.error(error.message);
         process.exit(1);
     }
-    console.log(`
+    global.console.log(`
```
Comment (djhi) [inline admin/bin/comfy-admin.js]: why the global ?

File: admin/package.json [hunk 2]
```diff
@@ -3,14 +3,15 @@
...
+    "babel-plugin-transform-react-inline-elements": "6.22.0",
     "babel-polyfill": "6.23.0",
+    "babel-preset-stage-0": "6.24.1",
     "fetch-ponyfill": "4.0.0",
-    "babel-preset-stage-0": "^6.24.1",
     "babel-runtime": "6.23.0",
     "freactal": "1.0.0",
+    "http-server": "0.10.0",
+    "ini": "1.3.4",
     "lodash.merge": "^4.6.0",
```
Comment (djhi) [inline admin/package.json]: there are lots of `^`

File: admin/bin/comfy-admin.js (hunk 1 above)
Comment (ThieryMichel) [inline admin/bin/comfy-admin.js]: eslint

Comment (ThieryMichel) [inline admin/bin/comfy-admin.js]: I could have added a `// eslint-disable-line`. But I prefer this solution.

Comment (djhi) [inline admin/bin/comfy-admin.js]: I prefer the eslint disable as it explicitly tell we want this console.log, but nvm :)

**Pull request merged**